STDatabase/src/commonTest/kotlin/com/infomaniak/multiplatform_swisstransfer/database/UploadControllerTest.kt

```diff
@@ -43,7 +43,7 @@ class UploadControllerTest {
     @Test
     fun canCreateAndGetAnUpload() = runTest {
         val dummyUpload = DummyUpload.uploads.first()
-        uploadController.insert(dummyUpload)
+        uploadController.insertAndGet(dummyUpload)
```

Comment:
Can we also check that the `Upload` we get from `insertAndGet` is the same as the one returned from `getUploadByUUID` ? Just to be sure. Also, the fact that we don't use a value we returned is not really good.